packages/core/package.json

```diff
@@ -26,6 +22,26 @@
       "module": "./dev/dist/xstate-dev.esm.js",
       "default": "./dev/dist/xstate-dev.cjs.js"
     },
+    "./actors": {
+      "module": "./actors/dist/xstate-actors.esm.js",
+      "default": "./actors/dist/xstate-actors.cjs.js"
+    },
+    "./actors/promise": {
+      "module": "./actors/promise/dist/xstate-actors-promise.esm.js",
+      "default": "./actors/promise/dist/xstate-actors-promise.cjs.js"
+    },
+    "./actors/reducer": {
+      "module": "./actors/reducer/dist/xstate-actors-reducer.esm.js",
+      "default": "./actors/reducer/dist/xstate-actors-reducer.cjs.js"
+    },
+    "./actors/callback": {
+      "module": "./actors/callback/dist/xstate-actors-callback.esm.js",
+      "default": "./actors/callback/dist/xstate-actors-callback.cjs.js"
+    },
+    "./actors/observable": {
+      "module": "./actors/observable/dist/xstate-actors-observable.esm.js",
+      "default": "./actors/observable/dist/xstate-actors-observable.cjs.js"
+    },
```

Comment:
What is the goal of those entries though? They are not even suggested by VS Code as a valid autoimport target:
<img width="534" alt="auto-import suggestion in the VS Code" src="https://user-images.githubusercontent.com/9800850/214584805-2418a4f4-2ac9-469a-a191-12a54cc7da4a.png">

I've done some testing and VS Code always filters out "duplicate" entries like this and prefers paths that are the shortest/closest to the root.

My initial fear was those duplicate entries would be displayed - which would be worse than the current situation. But even if they are not displayed there, I don't think this brings a lot of value to the table. Would those be documented anyhow? What we'd use in the docs examples? When one would like to use the longer ones?

Reply:
My motivation was for "import only what you need". Maybe there's a better way to accomplish that?

Reply:
> My motivation was for "import only what you need". Maybe there's a better way to accomplish that?

In a way, this doesn't accomplish this in full for actors since `fromObservable` and `fromEventObservable` live in the same entry :p

> That is, if I only invoke promises, I don't want to include the code for invoking observables, reducers, callbacks, etc. etc. (there might be more in the future)

This stuff is quite easily tree-shakeable. If you are not using a bundler then sure, you might pull some extra code into your runtime - but if that's a concern for somebody then they should use tooling. It's rather an impossible task for us to chunk everything into the smallest bits possible - we don't aim for 1 file-1 export structure.

Reply:
We can remove the entries but leave them in these files for organization, what do you think?

Reply:
Works for me

Reply:
I pushed out a commit with just that